### ISSUE TITLE
Feature request: Resume failed downloads

### ISSUE THREAD

**Issue by KridOCaign** [dteviot/WebToEpub#1136]
**Is your feature request related to a problem? Please describe.**
Yes.
If a chapter encounters an error, such as 521, then web2epub will terminate there and save what it has with no option to retry and continue.

**Describe the solution you'd like**
Include the option to resume an interrupted download with the chapter where the failure occurred.

**Describe alternatives you've considered**
Manually setting the download start to the failed chapter and then merging the parts after the fact was an option, but merging documents can have odd behavior at times which requires manual correction.

**Comment by kolbdog3333** [comment]
I would like this as well it would help especially if packing aborts in the middle of getting a story it was half way done getting the chapters then it aborted now it starts all over again

**Comment by Kiradien** [comment]
It's not a perfect solution, but you can actually import the incomplete epub into the "Library" and uncheck the existing chapters before clicking "Add to Library" - That will update the epub in the library to be downloaded with the other chapters. It will sort-of do what you want, but I agree that it would be nice to not need so many steps.

**Comment by mhussaincov94** [comment]
since web to epub uses your browser cache for storage of chapters before packing starts, would it not be easia to get chapters from cache swiftly then rezume fetching from the site?
is that possible?

**Comment by dteviot** [comment]
@KridOCaign 
>  would it not be easia to get chapters from cache swiftly then rezume fetching from the site

No.  It's not possible for the WebToEpub to directly access the browser's cache.

**Comment by mhussaincov94** [comment]
ok, my bad, sorry

**Comment by AJolly** [comment]
> It's not a perfect solution, but you can actually import the incomplete epub into the "Library" and uncheck the existing chapters before clicking "Add to Library" - That will update the epub in the library to be downloaded with the other chapters. It will sort-of do what you want, but I agree that it would be nice to not need so many steps.
@Kiradien  how do you import an incomplete epub?

**Comment by AJolly** [comment]
Sadly my current case it's definitely not downloading the incomplete ePub.
(And it doesn't seem to have automatically stored it in the extension
storage anywhere I can see)
